mriqc/interfaces/functional.py

```diff
@@ -95,13 +95,13 @@ class FunctionalQC(SimpleInterface):
     def _run_interface(self, runtime):
         # Get the mean EPI data and get it ready
         epinii = nb.load(self.inputs.in_epi)
-        epidata = np.nan_to_num(epinii.get_data())
+        epidata = np.nan_to_num(epinii.get_fdata())
         epidata = epidata.astype(np.float32)
```

Comment:
Here I think just using the dataobj will waste less memory (`get_fdata()` will cache the result and force to float64 even if there's no need). Also, might as well cast directly to float32. `nan`s will be `nan` in either type.

```suggestion
        epidata = np.nan_to_num(np.float32(epinii.dataobj))
```